### ISSUE TITLE
Translations of "mountain" missing

### ISSUE THREAD

**Issue by ChristianSi** [tatuylonen/wiktextract#108]
All the translations of the English word "mountain" seem to be missing. Wiktionary lists them on a [separate page](https://en.wiktionary.org/wiki/mountain/translations#Noun), but I can't find them in the [parsed entry](https://kaikki.org/dictionary/English/meaning/m/mo/mountain.html).

**Comment by tatuylonen** [comment]
The problem was that the translation subpage in Wiktionary (mountain/translations) was missing the language and part-of-speech subtitles.  I fixed this in Wiktionary and also added the {{translation subpage}} template at the beginning.  That change should get included within a month or so after the next Wiktionary database dump.

However I also implemented a kludge to address this directly in the extractor for better robustness and a faster fix.  Now if it fails to find the translations on the translation subpage under the correct language and part-of-speech, it will look for just a "Translations" section.  Since the translations subpage is only looked up if there is a reference to it on the main page, I don't expect this to cause problems.  I just committed this fix too, and with it, the issue should be fixed on https://kaikki.org after the next update (probably within the next couple of days; the update running right now does not yet include this change).
